megamek/src/megamek/common/Report.java

```diff
@@ -489,7 +487,7 @@ public String getText() {
             }
             //draw the sprite at the beginning of the line
             if(imageCode != null && imageCode.length() > 0){
-                if(text.substring(0,1).equals("\n")){
+                if(text.length() >= 2 && text.substring(0,1).equals("\n")){
```

Comment:
Is `text.startsWith("\n")` not an option? Simpler than a length check and first char check.

Reply:
Ah, good call. That flew right past me.